crates/aggchain-proof-builder/src/lib.rs

```diff
@@ -286,6 +289,20 @@ impl<ContractsClient> AggchainProofBuilder<ContractsClient> {
             }
         }
 
+        {
+            let aggchain_params_values = AggchainParamsValues::from(&fep_inputs);
+
+            info!("Aggchain-params unrolled values: {aggchain_params_values:?}");
+            info!(
+                "Aggchain-params abi encoded packed: {}",
+                hex::encode(fep_inputs.encoded_aggchain_params())
+            );
+            info!(
+                "Aggchain-params keccak-hashed: {}",
+                fep_inputs.aggchain_params()
+            );
```

Comment:
This will make a lot of noise in the logs. Should it be `trace`?

Reply:
very needed for now I think so `info` sounds reasonable to me to avoid having to say to everyone to enable the right log level